src/main/java/org/kohsuke/github/GHAppInstallations.java

```diff
@@ -0,0 +1,21 @@
+package org.kohsuke.github;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public class GHAppInstallations {
+
+    @JsonProperty("total_count")
+    private int totalCount;
```

Comment:
This is done automatically by Jackson. 
```suggestion
    private int totalCount;
```